src/AuthenticationPortal.Web/Controllers/SigninController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Authentication.Services;
+using AuthenticationPortal.Web.Validations;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Authentication.Controllers
+{
+    [Route("api/v1.0/ORP/[controller]")]
+    [ApiController]
+    public class SigninController : ControllerBase
+    {
+        private readonly IUserAuthentication _userAuthentication;
+        private readonly SignInRequestValidation _requestValidation = new SignInRequestValidation();
+        public SigninController(IUserAuthentication userAuthentication)
+        {
+            _userAuthentication = userAuthentication;
+        }
+
+        // GET: api/Signin       
+        [HttpGet]
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: api/Signin/5
+        [HttpGet("{id}", Name = "Get")]
+        public string Get(int id)
+        {
```

Comment:
did not get what all apis you are exposing under this route

Reply:
give appropriate names then

Reply:
Done